plugins/events-manager/buddypress/profile.php

```diff
@@ -383,6 +383,7 @@ class='widefat post fixed'>
 echo do_shortcode( '[cwp_notify_em_cat]' );
 $user_id     = get_current_user_id();
 $member_link = bp_core_get_userlink( $user_id, '', true );
+echo do_shortcode( '[contact-form-7 id="4014" title="Professional interest suggestion"]' );
```

Comment:
Any suggestions? I wish we had a theme options page, we could throw this type of stuff in there (along with rebranding etc like for TLDP)... so admin could add the right form ID. I could add an option to the customize menu, but that's under appearance so that doesn't make too much sense.. hmm

Reply:
sounds like you are leaning towards theme options. No arguments here.

Reply:
commit soon a solution for your consideration